src/script/api/game/game_window.hpp.sq

```diff
@@ -126,7 +126,7 @@ void SQGSWindow_Register(Squirrel *engine)
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_GAME_OPTIONS,                           "WC_GAME_OPTIONS");
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_CUSTOM_CURRENCY,                        "WC_CUSTOM_CURRENCY");
 	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_CHEATS,                                 "WC_CHEATS");
-	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_EXTRA_VIEW_PORT,                        "WC_EXTRA_VIEW_PORT");
+	SQGSWindow.DefSQConst(engine, ScriptWindow::WC_EXTRA_VIEWPORT,                        "WC_EXTRA_VIEWPORT");
```

Comment:
What is this string used for? Does it define part of the scripting API; and if so, what should we do about it?

Reply:
It does modify the script API, but it will only affect GS that do "tutorial" stuff with calling attention to specific windows or widgets.

Reply:
Worth adding to the compatibility scripts and changelog, do you think?

Reply:
I'm quite sure a total of zero scripts depend on this, but maybe someone could grep BaNaNaS for the string.

Reply:
> I'm quite sure a total of zero scripts depend on this, but maybe someone could grep BaNaNaS for the string.

I'll leave that up to somebody who knows how to do that :wink:

Reply:
Anyway you never modify .sq by hand. There are scripts to generate them.

Reply:
Not particularly relevant here, given how simple the change is...

Reply:
> Anyway you never modify .sq by hand. There are scripts to generate them.

It'd be very useful if there was a file somewhere with all these kinds of rules in it. If there is one, I haven't found it yet.

Besides that, if it's automatically generated, then why does it need to be under version control?

Reply:
It should be mentioned in CONTRIBUTING.md or COMPILING.md. If it's not... well, #8192 will fix it anyway.

Reply:
> It should be mentioned in CONTRIBUTING.md or COMPILING.md. If it's not... well, #8192 will fix it anyway.

I'm pretty sure I read those; although I can always reread them in case I missed or didn't comprehend something 😊